doc/releases/release-notes-2.7.rst

```diff
@@ -249,7 +249,7 @@ Bluetooth
 
 * Audio
 
-  * Added the Microphone Input Control Service and client.
```

Comment:
Don't modify old release notes please :)

Reply:
Release notes kept unchanged.

Reply:
It is still changed from 
```
Added the Microphone Input Control Service and client.
```
to
```
Added the Microphone Input Control Service Service and client.
```
(note the extra "Service")